pytket/extensions/qujax/qujax_convert.py

```diff
@@ -72,6 +72,11 @@ def _symbolic_command_to_gate_and_param_inds(
     :param symbol_map: ``dict``, maps symbolic pytket parameters following the order
         in this dict.
     :type symbol_map: dict
+    :param lambdify_gates: boolean that determines whether gate is represented by a
```

Comment:
Is this something that we should add to the example notebooks?
Have you thought about if there is a good way to detect this automatically, and not ask the user to give this parameter?

Reply:
Detecting this automatically is a good suggestion, it should be doable but I need to see how easy this is to check using sympy. 

In any case, I would keep the flag because there is a trade-off in lambdifying the gates or not (performance, as I mentioned earlier, but also in readability of the output of `tk_to_qujax_args`, which would also then no longer easily serialisable). 

So, what I now have in mind is that the default behaviour would be to detect the presence of symbols that have been manipulated, and have the user force the behaviour through the flag if they wish to.

If we manage to implement automatic detection of this, I would say it doesn't need to be in a notebook and can be instead just mentioned in a section of the documentation.